app/services/people/importer.rb

```diff
@@ -2,60 +2,89 @@
 
 module People
   class Importer
-    PROFILE_IDENTIFIERS_MAPPING = {
+    NOMIS_IDENTIFIER_MAPPING = {
       'police_national_computer' => :pnc_number,
       'prison_number' => :prison_number,
       'criminal_records_office' => :cro_number,
     }.freeze
 
-    attr_accessor :nomis_attributes
-
     def initialize(nomis_attributes)
-      self.nomis_attributes = nomis_attributes
+      @nomis_attributes = nomis_attributes
     end
 
     def call
       person = Person.find_or_initialize_by(nomis_prison_number: nomis_attributes[:prison_number])
+      person.assign_attributes(person_attributes)
+
       profile = person.latest_profile || person.profiles.build
-      profile.assign_attributes(attributes)
-      # return profile rather than person so that save does both profile and person if both are new
+      profile.assign_attributes(profile_attributes)
+
       profile
```

Comment:
Something isn't quite right here: this is the `People::Importer` but it is returning a `profile`.
If it is possible, I think we should return `person` rather than `profile` ?

Reply:
Although this is marked as resolved, I was wondering the same thing (maybe conversation happened offline)?

Reply:
hi @cwrw see this conversation: https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/512/files#r433164600  - there are further updates to resolve this in the future